Documentation/ApiOverview/DirectoryStructure/Index.rst

```diff
@@ -27,6 +27,24 @@ the paths within PHP code.
   - :Directory,20: Directory
     :Description,80: Description
 
+  - :Directory: :file:`_assets/`
+    :Description:
+      This directory includes symlinks to resources of extensions, as consequence
+      of this and further structure changes the folder :file:`typo3conf/ext/` is
+      not created or used anymore.  
+      So all files like CSS, JavaScript, Icons, Fonts, Images, etc. of extensions 
+      are not linked anymore directly to the extension folders but to the directory
+      :file:`_assets/`.
+      
+      .. note::
+        This directory :file:`_assets/` and the related changes depend on the
+        system-extension `composer_installers` in version 4+.
+        Previous versions of `composer_installers` used the classical directory
+        structure with :file:`typo3conf/ext/` for extensions.
+        
+        The system-extension `composer_installers` in version 4+ was created for
```

Comment:
That is not really true ;-) v4 is already a dependency in TYPO3 v11, but not installed as it is RC by now and not a stable version available:
https://github.com/TYPO3/typo3/blob/11.5/composer.json#L98
When v4.0.0 is released it can be also be used in TYPO3 v11 regularly (and will when using the not recommended "composer update" on your project. But now you can also install it only manually by requiring the RC version in your project's composer.json.

Reply:
No, it's the default option but no dependency.
This article shines a light on it: https://usetypo3.com/composer-changes-for-typo3-v11-and-v12.html

Reply:
```
        Previous versions of `typo3/cms-composer-installers` used the classical directory
        structure with :file:`typo3conf/ext/` for extensions.

        The composer-plugin `typo3/cms-composer-installers` in version 4+ was created for
```